sphinx/writers/html5.py

```diff
@@ -0,0 +1,934 @@
+# -*- coding: utf-8 -*-
+"""
+    sphinx.writers.html5
```

Comment:
Could you tell me how did you make this? This writer is too large to review whole of code. So I'd like to know the difference with html4 writer.
As a quick look, it is almost same as html4 writer. So I guess you were copied the html4 writer and rewrote the parent class. is this right?